docs/porting-guide.rst

```diff
@@ -138,73 +138,265 @@ In particular, when running on top of CPython, HPy will automatically provide
 a ``tp_dealloc`` which decrefs all the fields listed by ``tp_traverse``.
 
 
+Direct C API to HPy Mappings
+----------------------------
+
+In many cases, migrating to HPy is as easy as just replacing a certain C API
+function by the appropriate HPy API function. The table :ref:`table-mapping` a
+mapping between C API and HPy API functions. You can just apply this mapping
+without being cautious because this is a generated mapping which HPy also uses
+internally in the :term:`CPython ABI` mode.
+
+..  _table-mapping:
+
+.. table:: Safe API function mapping
+    :widths: auto
+
+    ================================================ ================================================
+    C API function                                   HPy API function
+    ================================================ ================================================
+    ``PyLong_FromLong``                              :c:func:`HPyLong_FromLong`
```

Comment:
This is great!

Maybe consider linking the CPython C API function to its documentation like
``` 
:py:func:`PyLong_FromLong`

Reply:
Done. I modified the autogen class such that it generates links to `https://docs.python.org/3/c-api/<PAGE>.html#c.<FUNCTION>` since `:py:func:...` would refer to Python functions that have been seen by some autodoc extension.